test/language/arrow-function/Arrow-Function_rules-for-prototype.js

```diff
@@ -0,0 +1,13 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 14.2
+description: >
+    Arrow functions are like functions, except they throw when using the
```

Comment:
This test seems a bit broader than this sentence implies. Not sure what a good description would be though...